frontend/packages/text-editor/src/TextEditor.tsx

```diff
@@ -100,6 +100,7 @@ export const TextEditor = ({
             color={ButtonColor.Primary}
             onClick={handleAddNewEntryClick}
             disabled={isFetchingTranslations}
+            data-testid='text-editor-btn-add'
```

Comment:
React Testing Library is trying to expose methods and utilities that encourage us to write tests that closely resemble how our web pages are used. React Testing Library has a [priority list for recommended queries ](https://testing-library.com/docs/queries/about#priority). 

The docs about test-id: 
_**getByTestId**: The user cannot see (or hear) these, so this is only recommended for cases where you can't match by role or text or it doesn't make sense (e.g. the text is dynamic)._

What about deleting the data-testid attribute and use `getByRole('button', { name: 'Ny tekst' }); ` within the unit-test instead?

```suggestion
        
```


The tests may run a little slower but the advantage of testing a11y overrides the disadvantage of slightly slower tests. 
I think we should follow the library's documentation to ensure that the correct elements are used. It will help us write semantic HTML. 

What do you think? :)

Reply:
ah, this test-id is perhaps used by cypress and not Rect Testing Library? Should we have a convention to make it easier to separate test-id between RTL and Cypress? What about do something like the suggestion? :) 
```suggestion
            data-cy-testid='text-editor-btn-add'
```

Reply:
Disagree. That would be over-engineering. I rather do that with shared enums.